elements/storytelling/src/main.js

```diff
@@ -1,12 +1,66 @@
 import { LitElement, html } from "lit";
+import markdownit from "markdown-it";
+import { unsafeHTML } from "lit/directives/unsafe-html.js";
+import { loadMarkdownURL } from "./helpers/index";
+import mainStyle from "../../../utils/styles/dist/main.style";
+const md = markdownit({ html: true });
```

Comment:
If we enable `html` from the start, then it would probably no harm in also adding some sanitation. E.g. using [DOMPurify](https://github.com/cure53/DOMPurify) - see https://github.com/markdown-it/markdown-it/blob/master/docs/security.md.
Or, we don't enable `html` but handle each different exception via plugins.

Reply:
Sure I thought I will do it when we start implementing things with actual markdown patterns. No issue I will add it right now.